nss-day-cohort-44/rare-rest-api-malicous-vortices
# Kn testing

# ISSUE TICKET 
## Description
Made posts list to filter by current logged in user if active=true in the url. 

## Type of change
- [ x] Bug fix (non-breaking change which fixes an issue)

# Testing Instructions
`git fetch origin Kn-testing `
`git checkout Kn-testing` 

User Instructions: 
if only running server side, on postman at http://localhost:8000/posts?active=true you should only see the posts made by the user whos token matched the one used in your auth header

# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
